lfs/transfer_queue.go

```diff
@@ -125,11 +127,17 @@ func (q *TransferQueue) processBatch() {
 
 	objects, err := Batch(transfers)
 	if err != nil {
-		q.errorc <- err
-		sendApiEvent(apiEventFail)
-		return
+		if isNotImplError(err) {
+			tracerx.Printf("queue: batch not implemented, disabling")
+			configFile := filepath.Join(LocalWorkingDir, ".gitconfig")
+			git.Config.UnsetLocalKey(configFile, "lfs.batch")
```

Comment:
It may make sense to set `lfs.batch` to `false` to override any potential global config values.